frontend/src/app/choreography/choreography.component.ts

```diff
@@ -55,6 +55,7 @@ export class ChoreographyComponent implements OnInit {
 
   animationIntervalId: number;
   frameInterval = 1000;
+  animationsOffCheckbox: boolean;
```

Comment:
**issue:** let's not call logical values with UI-related names -> take off the "checkbox" from the name